Core/HLE/sceDisplay.cpp

```diff
@@ -754,6 +754,9 @@ void __DisplayFlip(int cyclesLate) {
 	// Also let's always flip for animated shaders.
 	const ShaderInfo *shaderInfo = g_Config.sPostShaderName == "Off" ? nullptr : GetPostShaderInfo(g_Config.sPostShaderName);
 	bool postEffectRequiresFlip = false;
+
+	bool duplicateFrames = g_Config.bRenderDuplicateFrames && g_Config.iFrameSkip == 0;
```

Comment:
Dead store?  I assume this was meant to end up lower down in place of `g_Config.bRenderDuplicateFrames`?

-[Unknown]

Reply:
Eh oops, sigh. Thanks